### PR TITLE
LPD-15336 Add Picklist Object Field in Object Definition Node - Playwright Test

### PR DESCRIPTION
**Pull request by igor-franca**
Related [Ticket](https://liferay.atlassian.net/browse/LPD-15336)

### CODE REVIEW

**Comment by igor-franca** [comment]
The last pull aimed to resolve conflicts with master

**Comment by igor-franca** [comment]
ci:test:sf

**Review by cgmonte — COMMENTED** [review]
Hey @igor-franca, nice work! I just made a small suggestion. I think that always creating a single item picklist is not ideal, as other scenarios may require different picklists. Please read my related comment on this.

File: modules/test/playwright/helpers/ListTypeAdminApiHelper.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	async postRandomListTypeDefinition() {
+		const listTypeDefinitionExternalReferenceCode =
+			'ListTypeDefinition' + getRandomInt();
+
+		const listTypeEntryExternalReferenceCode =
+			'ListTypeEntry' + getRandomInt();
+
+		const requestBody = {
+			externalReferenceCode: listTypeDefinitionExternalReferenceCode,
+			listTypeEntries: [
```
[the remark this answers is not in the document]
Comment (igor-franca) [reply modules/test/playwright/helpers/ListTypeAdminApiHelper.ts:33]: Hey @cgmonte. I agree with you, I don't need to create a `listTypeEntries` here. But to maintain consistency with the objects example. I think we can make this post without entries.

And when necessary, create another function to post the entries of a list type definition.

I believe it is better than trying to add `listTypeEntries` logic within the `listTypeDefinition` post.

What do you say?

[the remark this answers is not in the document]
Comment (cgmonte) [reply modules/test/playwright/helpers/ListTypeAdminApiHelper.ts:33]: Yes, sure! No problem. Whenever we need to create an item, we can introduce that method, then later maybe a method that abstracts both list and items creation and do everything in one function call.

**Comment by igor-franca** [comment]
Hey @cgmonte, there are a test falling in source format related to another test, [see](https://liferay.slack.com/archives/CLD39UKM5/p1711025333221849)

I'm getting ahead of myself and will use a commit that solves this problem to ensure that with this solution and my new modifications we won't have any problems in SF

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:test:playwright

**Comment by cgmonte** [comment]
> Hey @cgmonte, there are a test falling in source format related to another test, [see](https://liferay.slack.com/archives/CLD39UKM5/p1711025333221849)
> 
> I'm getting ahead of myself and will use a commit that solves this problem to ensure that with this solution and my new modifications we won't have any problems in SF

No problem, someone else will probably merge a fix before this is forwarded :pray:

**Comment by igor-franca** [comment]
The next commit aims to remove the fix for the SF issue in the Journal Article, since this fix is ​​already in master.

**Review by cgmonte — COMMENTED** [review]
LGTM!

**Comment by igor-franca** [comment]
The correction in SF was added to master

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:forward:force

**Pull request closed without merging**